src/iOS/Xamarin.iOS/Samples/Map/GetElevationAtPoint/GetElevationAtPoint.cs

```diff
@@ -0,0 +1,193 @@
+                    _overlay.Graphics.Clear();
+
+                    // Get the elevation value.
+                    double elevation = await _baseSurface.GetElevationAsync(e.Location);
+
+                    // Set the text displaying the elevation.
+                    _elevationTextSymbol.Text = $"{Math.Round(elevation)} m";
+                    _elevationTextGraphic.Geometry = new MapPoint(e.Location.X, e.Location.Y, e.Location.Z + 850);
+
+                    // Add the text to the graphics overlay.
+                    _overlay.Graphics.Add(_elevationTextGraphic);
+
+                    // Add the marker indicating where the user tapped.
+                    _overlay.Graphics.Add(new Graphic(e.Location, _elevationMarker));
+                }
+            }
+            catch (Exception ex)
+            {
+                Debug.WriteLine(ex.Message);
+                CreateErrorDialog(ex.Message);
+            }
+            finally
+            {
+                // Re-add to the event handler.
+                _mySceneView.GeoViewTapped += SceneViewTapped;
+            }
+        }
+
+        public override void LoadView()
+        {
+            // Create the views.
+            View = new UIView();
+
+            _mySceneView = new SceneView();
+            _mySceneView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+            UILabel helpLabel = new UILabel
+            {
+                Text = "Tap to find the elevation for a point.",
+                BackgroundColor = UIColor.FromWhiteAlpha(0f, .6f),
+                TextColor = UIColor.White,
+                TextAlignment = UITextAlignment.Center,
+                TranslatesAutoresizingMaskIntoConstraints = false
+            };
+
+            // Add the views.
+            View.AddSubviews(_mySceneView, helpLabel);
+
+            // Lay out the views.
+            NSLayoutConstraint.ActivateConstraints(new[]{
+                _mySceneView.TopAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.TopAnchor),
+                _mySceneView.BottomAnchor.ConstraintEqualTo(View.BottomAnchor),
+                _mySceneView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                _mySceneView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+
+                helpLabel.TopAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.TopAnchor),
+                helpLabel.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                helpLabel.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+                helpLabel.HeightAnchor.ConstraintEqualTo(40)
+            });
+        }
+
+        public override void ViewDidLoad()
+        {
+            base.ViewDidLoad();
+            Initialize();
+        }
+
+        public override void ViewWillAppear(bool animated)
+        {
+            base.ViewWillAppear(animated);
+
+            // Handle taps on the scene view for getting elevation.
+            _mySceneView.GeoViewTapped += SceneViewTapped;
+        }
+
```

Comment:
Don't forget to add the `ViewDidDisappear` override with the corresponding event unsubscription, `_mySceneView.GeoViewTapped -= SceneViewTapped`.